### DISCUSSION TITLE
Confusion about the Galactic Longitude

### DISCUSSION THREAD

**Discussion by Arcturios** [Stellarium/stellarium#2286] [General]
I was looking up the Galactic coordinates 

It's giving negative degrees for some placements

How do I determine the angular separation between 
Sun: - 28°17'22.2"  and Eris: 151°31'07.6

 Quaoar - 28°07'38.4"
I can easily tell that Sun and Quaoar are in conjunction with an orb of 10 minutes of arc.

How come the Galactic Longitude has negative degrees?

I thought Galactic longitude is 0° - 360°

Is there a way to change it to that?

If not, I hope that there will be some changes to allow people to use the  0° - 360°

**Comment by alex-w** [comment]
Please check latest beta - we fixed this issue

**Reply by Arcturios** [reply]
Where on the site can I find the latest beta?

I am sorry if I come off stupid to you.
This is my first time on this site. 

I really love and appreciate your program. 
I am just an  astrononomical enthusiast who is also very much into Astrology which I approach quite multi-dimensionally and geometrically.
One of my main interests is Pluto's fellow transneptunian dwarf planet/candidates.

**Comment by gzotti** [comment]
I am surprised that you would even care for Galactic coordinates of minor bodies.
Juts like on Earth, You can count longitudes 0...360 or -180...180. Days ago we amended our -180...180 system to use the official 0...360 system, and this will appear in the next release. But doing the conversion math is really trivial for a second-grader.

**Reply by Arcturios** [reply]
Well...I do care for them.
to each, his/her own

I have been interested in the transneptunian objects since the discovery of Varuna back in 2001, and I like looking at them in any coordinate system. 

I am a neurodivergent with Dyslexia, Dyspraxia, ADHD.

**Reply by astroscot** [reply]
There is absolutely no need for this arrogant and unhelpful reply  (gzotti) to a perfectly reasonable question from Arcturios. If you can't reply in a civil and helpful manner then don't.
